### PR TITLE
Wide Array of Fixes

### PR DESCRIPTION
**Pull request by ronhowe**
Creature Defenses
Ahch-To and Takodona Icons
Errata
etc.

### CODE REVIEW

File: Dark.json [hunk 1]
```diff
@@ -54719,7 +54728,7 @@
         ],
         "lightSideIcons": 1,
         "darkSideIcons": 2,
-        "gametext": "Dark:  Once per game, if you occupy three battlegrounds, may retrieve a Black Sun agent into hand..  Light:  Once per game, if you control, may retrieve [Reflections II] Dash into hand."
```
Comment (scottrick) [inline Dark.json:54722]: Is this an errata to this card?  Looking in scomp link, the card image does not match the new text.

Comment (ronhowe) [reply Dark.json:54722]: Re-downloaded the Set 9 PDF and looked at the current state.  You're right that this game text is wrong.

I will submit an updated PR tonight.

Comment (ronhowe) [reply Dark.json:54722]: Oh wait.  The red line is the old line.  Per the Set 9 PDF, text was errata'd.
image

Comment (scottrick) [reply Dark.json:54722]: It seems there may be other issues then, I just checked card at random.

Comment (scottrick) [reply Dark.json:54722]: Okay, so the scomp link images are out of date then.  Got it.

**Comment by thomasmarlin** [comment]
Looks good.  I noticed that all of the objectives include the objective-title in the gametext field. I'm guessing this for and from Holotable CDFs. At some point, I'd like to get that removed, but it's pretty low on my priority list.

Thanks for the taking the time to do all of these!

**Pull request merged**